spaces/domains.json

```
@@ -217,6 +217,7 @@
   "vote.moonpirate.finance": "moonpirate.eth",
   "vote.mai.finance" : "qidao.eth",
   "dao.modefi.io" : "mod.eth",
+  "vote.conqueruranus.eth":"conqueruranus.eth"
```

A comma is missing at the end of the line of your sub-domain.
```suggestion
  "vote.conqueruranus.XXX": "conqueruranus.eth",
```
Are you sure that your main domain is http://conqueruranus.eth/?